packages/main/src/SplitButton.js

```diff
@@ -0,0 +1,449 @@
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		/**
+		 * Indicates if there is action on default or arrow button triggered by keyboard
+		 * (Space, Enter, F4, Arrow Up/Down, Alt+Arrow Up/Down)
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 */
+		_keyboardAction: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		/**
+		 * Indicates if there is SHIFT or ESCAPE key pressed
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 */
+		_shiftOrEscapePressed: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		/**
+		 * Defines the icon of the text button
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @private
+		 */
+		 _textButtonIcon: {
+			type: String,
+			defaultValue: "",
+			noAttribute: true,
+		},
+
+		/**
+		 * Defines the active state of the text button
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 */
+		 _textButtonActive: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		/**
+		 * Defines the active state of the arrow button
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 */
+		 _arrowButtonActive: {
+			type: Boolean,
+			noAttribute: true,
+		},
+	},
+	events: /** @lends sap.ui.webcomponents.main.SplitButton.prototype */ {
+		/**
+		 * Fired when the user clicks on the default action.
+		 */
+		"click": {},
+
+		/**
+		 * Fired when the user clicks on the arrow action.
+		 */
+		"arrow-click": {},
+	 },
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * <code>ui5-split-button</code> enables users to trigger actions. It is constructed of two separate actions -
+ * default action and arrow action that can be activated by clicking or tapping, or by
+ * pressing certain keyboard keys - <code>Space</code> or <code>Enter</code> for default action,
+ * and <code>Arrow Down</code> or <code>Arrow Up</code> for arrow action.
+ *
+ * <h3>Usage</h3>
+ *
+ * <code>ui5-split-button</code> consists two separate buttons:
+ * <ul>
+ * <li>for the first one (default action) you can define some <code>text</code> or an <code>icon</code>, or both.
+ * Also, it is possible to define different icon for active state of this button - <code>activeIcon</code>.</li>
+ * <li>the second one (arrow action) consists only <code>slim-arrow-down</code> icon.</li>
+ * </ul>
+ * You can choose a <code>design</code> from a set of predefined types (the same as for ui5-button) that offer
+ * different styling to correspond to the triggered action. Both text and arrow actions have the same design.
+ * <br><br>
+ * You can set the <code>ui5-split-button</code> as enabled or disabled. Both parts of an enabled
+ * <code>ui5-split-button</code> can be pressed by clicking or tapping it, or by certain keys, which changes
+ * the style to provide visual feedback to the user that it is pressed or hovered over with
+ * the mouse cursor. A disabled <code>ui5-split-button</code> appears inactive and any of the two buttons
+ * cannot be pressed.
+ * <br><br>
```

Comment:
You can also add Keyboard Handling section, I think the Split Button has some specific stuff that the arrow keys trigger the arrow-click action and that is nice to be documented.